libflux/src/flux/semantic/flatbuffers/semantic_generated.rs

```diff
@@ -12478,6 +12405,111 @@ pub mod fbsemantic {
         }
     }
 
+    pub enum DurationOffset {}
+    #[derive(Copy, Clone, Debug, PartialEq)]
+
+    pub struct Duration<'a> {
+        pub _tab: flatbuffers::Table<'a>,
+    }
+
+    impl<'a> flatbuffers::Follow<'a> for Duration<'a> {
+        type Inner = Duration<'a>;
+        #[inline]
+        fn follow(buf: &'a [u8], loc: usize) -> Self::Inner {
+            Self {
+                _tab: flatbuffers::Table { buf: buf, loc: loc },
+            }
+        }
+    }
+
+    impl<'a> Duration<'a> {
+        #[inline]
```

Comment:
_(I see this is mostly just moved code, but would like to ask this, just in case you know)_

Do we have a rule for when we expect inline attributes to be worth using? What heuristic do we use to balance compile time, binary size, and execution speed?

Reply:
Hmm...I'm not super familiar with this, so I'm not quite sure. I'd need to read up a bit on what inline does.

Reply:
I _completely_ missed that this was generated code. Please ignore me. I had thought that we were putting those annotations in.